api/types/saml.go

```diff
@@ -724,7 +569,7 @@ func (*teleportSAMLConnectorMarshaler) UnmarshalSAMLConnector(bytes []byte, opts
 			}
 		}
 
```

Comment:
This seems like a typo, or the previous iteration of `CheckAndSetDefaults` caused issues here. This should no longer be the case.